staging/src/k8s.io/apimachinery/pkg/util/version/version.go

```diff
@@ -183,8 +183,8 @@ func (v *Version) WithPreRelease(preRelease string) *Version {
 	return &result
 }
 
-// WithBuildeMetadata returns copy of the version object with requested buildMetadata
-func (v *Version) WithBuildeMetadata(buildMetadata string) *Version {
```

Comment:
this loses backward-compatibility from staging area.. but it's probably fine b/c no public repos in github is referencing the method.